### ISSUE TITLE
Virtualize the Icon Browser list

### ISSUE THREAD

**Issue by joebochill** [brightlayer-ui/doc-it#600]
#### Describe the desired behavior
The IconBrowser page should only render enough icons to fill the screen at one time.

#### Describe the current behavior
Right now all of the icons are rendered (>2000) and it causes poor performance on the Icon Browser page

#### Is this request related to a current issue?
Performance issues

#### Additional Context

There should be a number of third party libraries (or something in MUI) that we can use to do this.

**Comment by huayunh** [comment]
(This might be a MUI v5 thing; the existing docit icons page loads within a tolerable time)

**Comment by joebochill** [comment]
Yes, this is post-MUI5 conversion. The number of icons (DOM nodes) on the page has nearly doubled with the addition of the new icons.
